From 50ea5dd4dd2b121abd599b6475f95805822bb110 Mon Sep 17 00:00:00 2001
From: Olivier Roussel <olivier.roussel@inria.fr>
Date: Thu, 16 Jan 2025 17:13:00 +0100
Subject: [PATCH] Fix bindings cmake config for missing simpleApi

---
 bindings/Sofa/Bindings.SofaConfig.cmake.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/bindings/Sofa/Bindings.SofaConfig.cmake.in b/bindings/Sofa/Bindings.SofaConfig.cmake.in
index 8648921..f38f984 100644
--- a/bindings/Sofa/Bindings.SofaConfig.cmake.in
+++ b/bindings/Sofa/Bindings.SofaConfig.cmake.in
@@ -14,6 +14,7 @@ find_package(Sofa.Simulation.Core QUIET REQUIRED)
 # Required by Bindings.Sofa.Core
 find_package(Sofa.Component.Visual QUIET REQUIRED)
 find_package(Sofa.Component.Collision.Response.Contact QUIET REQUIRED)
+find_package(Sofa.SimpleApi QUIET REQUIRED)
 
 # Required by Bindings.Sofa.Simulation
 find_package(Sofa.Simulation.Graph QUIET REQUIRED)
-- 
2.34.1

